### ISSUE TITLE
How to train a new model

### ISSUE THREAD

**Issue by szk1509** [autonomousvision/differentiable_volumetric_rendering#48]
Hi, first of all I want to thank you for sharing your work :smiley:  on the other hand, after several attempts, I was wondering what I was missing/misinterpreting on the default.yaml file to train a new model.

The **structure of my dataset** looked like this:
- DataSetX/
   - ModelCad/
      - _info.yml_
      - _obj_Model01.ply_
      - ...
      - _obj_ModelXX.ply_
   - TrainSet/
      - Model01/
         - rgb/
            - _img00.jpg_
            - ...
            - _imgXX.jpg_
         - _info.yml_
      - ...
      - ModelXX/
   - TestSet/

The **config.yaml**:
data:
  dataset_name: DataSetX
  path: _data/DataSetX_
  ...
  input_type: image
  ...
  n_views: 0
  n_views_input: 0
  pointcloud_chamfer_file: pointcloud.npz
  ...
  img_folder: image
  img_folder_input: img_choy2016
  ...
  img_extension: png
  img_extension_input: jpg
  ...
  train_split: train
  val_split: val
  test_split: test
  ...

Questions:
1. What are the differences btw. img_folder and img_folder_input?
After modifying both values independently I still got the following error:
"Error occurred when loading field img of model rgb (05)"

2. Would you recommend to change the number of views value? or it is not strictly necessary?

3. Since my training set, validation set and cad models are separated in folders, do you consider it easier to change the structure to the structure presented by shapenet?

Thank you for your help,
Lizeth

**Comment by m-niemeyer** [comment]
Hi,

I would suggest you download one of our datasets (e.g. like you mentioned the ShapeNet one), and you write a script which converts the data from your format to our format. Then it should be much easier to adapt the config to your needs - you rather change small things like the image folder name or the image extension in the config, but not structural things like what you mention.

Good luck!

**Comment by daviszhu88** [comment]
I got the same problem when trying the train.py, with DTU data. I really think there should be something work that people can reproduce the results.
